reports/report_builder.py

```diff
@@ -2025,128 +2025,102 @@ def ws_60(self, ws):
     def ws_61(self, ws):
         """
         Cols: Sex of subject
-        Rows: Country, is_quoted
-        :: Show all countries
+        Rows: is_quoted
         :: Internet media type only
         """
-        r = 6
-        self.write_col_headings(ws, GENDER)
-
         counts = Counter()
         model = person_models.get('Internet')
-        for code, country in self.countries:
-            rows = model.objects\
-                    .values('sex', 'is_quoted')\
-                    .filter(**{model.sheet_name() + '__country':code})\
-                    .annotate(n=Count('id'))
+        
+        rows = model.objects\
+                .values('sex', 'is_quoted')\
+                .filter(**{model.sheet_name() + "__country__in": self.country_list}) \
+                .annotate(n=Count('id'))
 
-            rows = self.apply_weights(rows, model.sheet_db_table(), "Internet")
-            counts = {(row['sex'], row['is_quoted']): row['n'] for row in rows}
+        rows = self.apply_weights(rows, model.sheet_db_table(), "Internet")
+        for row in rows:
+            counts[row['sex'], row['is_quoted']] += row['n']
```

Comment:
Just out of curiosity, why are we updating `counts` here differently from the usual `counts.update`?

Reply:
Mainly just for readability. I did this while trying to understand what was happening. I'll revert it back for consistency.

Reply:
@kilemensi  Also, I had noticed some differences between the 2 approaches. 
For example with the 
`counts = {(row['sex'], row['is_quoted']): row['n'] for row in rows}` approach, the sum of the counts doesn't add up to the `model.objects` count. 

Here is an example on sheet 62
```
counts = Counter()
counts1 = Counter()

for row in rows:
    counts[TOPIC_GROUPS[row['topic']], row['equality_rights']] += row['n']
counts1 = {(TOPIC_GROUPS[row['topic']], row['equality_rights']): row['n'] for row in rows}

counts1 == counts # returns false
sum([c for c in counts.values()]) == 4703
sum([c for c in counts1.values()]) == 918
```
The total value in the database is `4703`.
Unless I'm missing something, we might have been returning incorrect values.

Reply:
Does calling `count.update` make any difference @esirK?  i.e. `counts.update({(r["topic"], r["equality_rights"]): r["n"] for r in rows})`

Reply:
No, it doesn't.

Reply:
I've noticed this only happens when we are using dict comprehension for the `row`. Looping through the rows normally and doing the `count.update` works fine.